exercises/change/example.js

```diff
@@ -71,12 +72,14 @@ export default class Change {
     };
 
     // is everthing searched?
-    const isDone = () => candidates.every(
-      candidate => isNumber(candidate) || candidate.isSearched());
+    const isDone = () => {
+      candidates.every(candidate => isNumber(candidate) || candidate.isSearched());
+    };
 
     // get the next unsearched member of the candidate array
-    const getNext = () => candidates.find(
-      candidate => !isNumber(candidate) && !candidate.isSearched());
+    const getNext = () => {
+      candidates.find(candidate => !isNumber(candidate) && !candidate.isSearched());
+    };
```

Comment:
OK. The reason build is failing here is because your code behaves differently from before. The thing without `{}` used to implicitly return the value - not anymore after adding `{}`. Hence `undefined` error in CI.

Either you can add a `return` there - or (better) you can change `{}` to `()`. Round brackets do implicit return and behave same as the code before.

Reply:
Thanks so much for the feedback @tejasbubane ! Really appreciate it 🙏👍😌
I'll take care of this tonight and then grab another one now that I know the routine 😄
Have a nice Sunday!

Reply:
Done 😃 I fixed the typo and learned something new today, awesome! 🎉